Triphouse-project/index.html

```diff
@@ -30,6 +30,64 @@ <h1 class="offer_title">What to be offer</h1>
                 </div>
             </div>
         </div>
-    </section>
+    </main>
+    <footer class="footer">
+        <div class="footer-content">
+            <div class="footer-content__list">
+                <ul>
+                    <h4><a href="#">Property types</a></h4>
+                        <li>
+                            <a href="#">How Triphouse works</a>
+                        </li>
+                        <li>
+                            <a href="#">Careers</a>
+                        </li>
+                        <li>
+                            <a href="#">Privacy</a>
+                        </li>
+                        <li>
+                            <a href="#">Terms</a>
+                        </li>
+                </ul>
+            </div>
+            <div class="footer-content__list">
+                <ul>
+                    <h4><a href="#">Property types</a></h4>
+                    <li>
+                        <a href="#">Guest houses</a>
+                    </li>
+                    <li>
+                        <a href="#">Hotels</a>
+                    </li>
+                    <li>
+                        <a href="#">Apartments</a>
+                    </li>
+                    <li>
+                        <a href="#">Vills</a>
+                    </li>
+                    <li>
+                        <a href="#">Holiday homes</a>
+                    </li>
+                    <li>
+                        <a href="#">Hostes</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="footer-content__list">
+                <ul>
+                    <h4><a href="#">Support</a></h4>
+                    <li>
+                        <a href="#">Contact Customer Service</a>
+                    </li>
+                    <li>
+                        <a href="#">FAQ</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="footer__info">
+            &copy; 2020 Triphouse, Inc. All rights reserved
```

Comment:
The text should wrap tag `p, span, a and similar`.

Reply:
fixed. Adding text to <p>